ui/components/component-library/avatar-network/avatar-network.tsx

```diff
@@ -77,16 +81,20 @@ export const AvatarNetwork: AvatarNetworkComponent = React.forwardRef(
                 aria-hidden="true"
               />
             )}
-            <img
-              className={
-                showHalo
-                  ? 'mm-avatar-network__network-image--size-reduced'
-                  : 'mm-avatar-network__network-image'
-              }
-              onError={handleOnError}
-              src={src}
-              alt={(name && `${name} logo`) || 'network logo'}
-            />
+            {isIconSrc(src) ? (
+              <Icon name={src} />
+            ) : (
```

Comment:
Added support for the Avatars to take in our Icons and not just remote images.

Reply:
I think this is what `<AvatarIcon />` is for?

Reply:
Great callout! After discussing, we don't need a change at this DS component level, reverting this file change and will handle the icon elsewhere.